### ISSUE TITLE
The environment name should be the first prefix

### ISSUE THREAD

**Issue by pherbel** [MicrosoftDocs/cloud-adoption-framework#271]
Read more our about it in our blog post: [Azure naming conventions](https://medium.com/gettech-blog/azure-naming-conventions-98d6e5a6ae80)

TL; DR
Miss the environment where you are doing actions it can cause big mistakes.
So, we prefer to move the environment name to the first place.
The brain can focus better on the prefix instead of the middle of the word.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c6dff4c3-d2d7-3825-d753-fc7338004f43
* Version Independent ID: b2c190ac-1d2e-93fc-caee-d59c278f549a
* Content: [Recommended naming and tagging conventions - Cloud Adoption Framework](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/naming-and-tagging)
* Content Source: [docs/ready/azure-best-practices/naming-and-tagging.md](https://github.com/MicrosoftDocs/cloud-adoption-framework/blob/master/docs/ready/azure-best-practices/naming-and-tagging.md)
* Service: **cloud-adoption-framework**
* Sub-service: **ready**
* GitHub Login: @BrianBlanchard
* Microsoft Alias: **brblanch**

**Comment by JanetCThomas** [comment]
Thanks for reporting - this issue is under review.  This is a Microsoft Internal DevOps Bug ID AB#286141

**Comment by alexbuckgit** [comment]
We've revised the structure of our naming guidance and encourage readers to choose the order of the components in resource names based on their organizational needs. Thanks for the feedback!
